hera_mc/cm_hookup.py

```diff
@@ -130,13 +118,20 @@ def get_hookup_from_db(self, hpn, pol, at_date, exact_match=False, hookup_type=N
         Parameters
         -----------
         hpn : str, list
-            List/string of input hera part number(s) (whole or first part thereof)
+            List/string of input hera part number(s) (whole or 'startswith')
+            If string
+                - 'cache' returns the entire cache file
```

Comment:
is 'cache' actually allowed in this method? It looks like this docstring was copied from `get_hookup` but that one explicitly handles 'cache' and this one doesn't.

Also, while you're re-arranging code, any function or method that is called by another function or method should come **before** the function or method that is calling it. So these two methods should be swapped.

Reply:
Good catch!  I've swapped the order.